x-pack/plugins/apm/public/components/shared/time_comparison/use_time_range_comparison.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { EuiTheme } from 'src/plugins/kibana_react/common';
+import { getDateDifference } from '../../../../common/utils/formatters';
+import { useTheme } from '../../../hooks/use_theme';
+
+function getChartTheme(theme: EuiTheme) {
+  return {
+    areaSeriesStyle: {
+      area: {
+        fill: theme.eui.euiColorLightestShade,
+        visible: true,
+        opacity: 1,
+      },
+      line: {
+        stroke: theme.eui.euiColorMediumShade,
+        strokeWidth: 1,
+        visible: true,
+      },
+      point: {
+        visible: false,
+      },
+    },
+  };
+}
+
+export function useTimeRangeComparison({
```

Comment:
Afaict this doesn't need to be a hook but can be a normal function if you pull `useTheme` out and call it separately.